common/ethics/00_ethics.txt

```diff
@@ -1958,7 +2111,7 @@ ethic_fanatic_materialist = {
 	
 	country_modifier = {
 		planet_jobs_upkeep_mult = -0.10
-		country_physics_tech_research_speed = 0.20
```

Comment:
This leaves physics with no associated ethic. I'd prefer to center trade bonuses on the prosperity tree as well to differentiate the themes and allow for a bit more mixing and matching.

Reply:
Will address in other convo I just unresolved. This one is outdated since current materialist ethic bonus in my branch is +districts -job upkeep